src/Initializer/Parameters/ModelParameters.cpp

```diff
@@ -34,6 +34,29 @@ ITMParameters readITMParameters(ParameterReader* baseReader) {
       itmEnabled, itmStartingTime, itmDuration, itmVelocityScalingFactor, reflectionType};
 }
 
+DamagedElasticParameters readDamagedElasticParameters(ParameterReader* baseReader){
+    auto* reader = baseReader -> readSubNode("equations");
+    const auto epsInitxx = reader->readWithDefault<real>("epsinitxx", 3.7986e-4);
+    const auto epsInityy = reader->readWithDefault<real>("epsinityy", -1.0383e-3);
+    const auto epsInitzz = reader->readWithDefault<real>("epsinitzz", -1.0072e-3);
+    const auto epsInitxy = reader->readWithDefault<real>("epsinitxy", 1.0909e-3);
+    const auto epsInityz = reader->readWithDefault<real>("epsinityz", 0);
+    const auto epsInitzx = reader->readWithDefault<real>("epsinitzx", 0);
+    const auto beta_alpha = reader->readWithDefault<real>("betaalpha", 0.05);
+    const auto aB0 = reader->readWithDefault<real>("aB0", 7.43e9);
+    const auto aB1 = reader->readWithDefault<real>("aB1", -12.14e9);
+    const auto aB2 = reader->readWithDefault<real>("aB2", 18.93e9);
+    const auto aB3 = reader->readWithDefault<real>("aB3", -5.067e9);
```

Comment:
Also here... Is there any justification for the choice of numbers?

Reply:
@Aangniu, could you please shed some light on how these numbers are chosen?

Reply:
These are chosen according to previous publications on this model. The basic idea is that these moduli (for granular phase) should be lower than the elastic moduli. We can fix them as default.

Reply:
Modified